From 98bc5d0405fe160cf72cb211de93ac2b5fa46474 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Fri, 5 Sep 2014 01:33:01 -0400
Subject: [iommu] vt-d: use defined macro instead of hardcoding

Message-id: <20140905013301.30867.43489.stgit@gir.stowe>
Patchwork-id: 91283
O-Subject: [RHEL7.1 PATCH 19/28] iommu/vt-d, trivial: use defined macro instead of hardcoding
Bugzilla: 1129808
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1129808
Upstream Status: 2fe2c6025d6a4939ae2fc97d1d761fc4a8d1abd9

commit 2fe2c6025d6a4939ae2fc97d1d761fc4a8d1abd9
Author: Jiang Liu <jiang.liu@linux.intel.com>
Date:   Mon Jan 6 14:18:17 2014 +0800

    iommu/vt-d, trivial: use defined macro instead of hardcoding

    Use defined macro instead of hardcoding in function set_ioapic_sid()
    for readability.

    Signed-off-by: Jiang Liu <jiang.liu@linux.intel.com>
    Signed-off-by: Joerg Roedel <joro@8bytes.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/iommu/intel_irq_remapping.c b/drivers/iommu/intel_irq_remapping.c
index 10d3187..fdf5753 100644
--- a/drivers/iommu/intel_irq_remapping.c
+++ b/drivers/iommu/intel_irq_remapping.c
@@ -324,7 +324,7 @@ static int set_ioapic_sid(struct irte *irte, int apic)
   return -1;
  }
 
- set_irte_sid(irte, 1, 0, sid);
+ set_irte_sid(irte, SVT_VERIFY_SID_SQ, SQ_ALL_16, sid);
 
  return 0;
 }
-- 
1.7.1